api/src/main/scala/hmda/api/http/ValidationErrorConverter.scala

```diff
@@ -50,9 +50,17 @@ trait ValidationErrorConverter {
 
   val editDescriptions = EditMetaDataLookup.values
 
-  private def rowDetail(err: ValidationError): RowEditDetail = {
+  private def editDetail(err: ValidationError, vs: HmdaFileValidationState): EditResultRow = {
+    val row = relevantRow(err, vs)
+
+    if (err.ts) EditResultRow(RowId("Transmittal Sheet"), relevantFields(err, row))
```

Comment:
I think I disagree with this one. In this case, `row` is a `HmdaFileRow` (i.e. a TS or a LAR). It doesn't just "expand" to the types of its parameters, since the `relevantRow` method has complex logic built in.